src/styles/blocks/catalog.scss

```diff
@@ -7,19 +7,27 @@
   padding: 50px 0;
 
   &__card {
+    position: relative;
     display: flex;
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
 
     width: 200px;
     height: 300px;
-    border: 1px solid #ccc;
+    border: 1px solid $card-border-color;
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color 300ms, transform 300ms;
```

Comment:
also would be great to move `300ms` to a variable because you use it several times